agents/agent_engine/tutorial_get_started_with_code_execution.ipynb

```
@@ -0,0 +1,2072 @@
+      "outputs": [],
+      "source": [
+        "sandbox_operation = client.agent_engines.sandboxes.create(\n",
+        "    name=agent_engine.api_resource.name,\n",
+        "    config=types.CreateAgentEngineSandboxConfig(display_name=\"my_first_sandbox\"),\n",
+        "    spec={\"code_execution_environment\": {}},\n",
+        ")\n",
+        "\n",
+        "# Save the sandbox resource name for later use\n",
+        "sandbox_resource_name = sandbox_operation.response.name\n",
+        "\n",
+        "print(\"✅ Sandbox created successfully!\")\n",
```

Ruff (0.14.3)

Use ternary operator `code_to_execute = code_match.group(1) if code_match else claude_response` instead of `if`-`else`-block

Replace `if`-`else`-block with `code_to_execute = code_match.group(1) if code_match else claude_response`

(SIM108)